core/templates/dev/head/components/CorrectEditorDirective.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the correct editor.
+ */
+
+// This directive controls an editor for selecting whether the answers
+// which fall under this answer group are correct or not. It also includes
+// 'Cancel' and 'Save' buttons which call respective 'onCancelCorrectEdit'
```

Comment:
... which call 'onCancelCorrectEdit' and 'onSaveCallback' callbacks respectively when clicked.

Reply:
I have removed onCancelCorrectEdit function because it was not required.(Updated the comment accordingly)